apps/web/modules/ee/license-check/lib/utils.ts

```
@@ -368,7 +370,21 @@
   }
   const licenseFeatures = await getLicenseFeatures();
   if (!licenseFeatures) return false;
-  return licenseFeatures.sso;
+  return true;
+  // return licenseFeatures.sso;
+};
+
+export const getIsSamlSsoEnabled = async (): Promise<boolean> => {
+  if (E2E_TESTING) {
+    const previousResult = await fetchLicenseForE2ETesting();
+    return previousResult && previousResult.features
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)